packages/translate/src/index.js

```diff
@@ -166,10 +166,12 @@ Translate.prototype.detect = function(input, callback) {
  *
  * @resource [Discover Supported Languages]{@link https://cloud.google.com/translate/v2/using_rest#supported-languages}
  *
+ * @param {options=} options - Configuration options.
+ * @param {options} options.target - Configuration options.
```

Comment:
Let's just accept a string that will be used as the target language:

```diff
* @param {string=} target - The language you intend to translate text to.
```

^ Feel free to re-word... just trying to be overly-helpful to the reader.